modules/packages/ZMQ.chpl

```diff
@@ -890,8 +890,15 @@ module ZMQ {
         // conditionally have ZeroMQ free the memory.
         // 
         // Note: the string factory below can throw DecodeError
+        // Another note: The bytes copies are parallelized in this file
+        // only temporarily, where long term we would like to do that
+        // parallelization in the internal bytes module, but are having
+        // to delay that due to some resolution issues. Also, the string
+        // assignment can likely be parallelized similarly, but was not
+        // included in this temporary effort.
```

Comment:
I might move this comment to the `parallelCreateBytesWithNewBuffer`. My thinking is that's the common place, and just commenting here could leave future readers wondering about the call in `recv`